rialto_airflow/dags/publish_orcid.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+from airflow.decorators import dag, task
+from airflow.models import Variable
+
+from rialto_airflow.mais import (
+    get_token,
+    current_orcid_users,
+    get_orcid_stats,
+)
+
+data_dir = Variable.get("data_dir")
+mais_base_url = Variable.get("mais_base_url")
+mais_client_id = Variable.get("mais_client_id")
+mais_client_secret = Variable.get("mais_secret")
+
+
+@dag(
+    # schedule=@weekly,
+    start_date=datetime.datetime(2024, 1, 1),
+    catchup=False,
+)
+def publish_orcid():
+    # @task()
```

Comment:
Why are parts of the code commented out? If we're not yet using it I suggest we remove it for now and add it back when it is needed.